CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Throw a proper exception when operating on a non-existing field with the dynamic API (#3292).
 * `DynamicRealmObject.setList` should only accept `RealmList<DynamicRealmObject>` (#3280).
+* Fixed a concurrency crash which might happen when `Realm.executeTransactionAsync()` called with a `OnSuccess` and Realm closed before `OnSuccess` called.
```

Comment:
Fixed a concurrency crash which might happen when `Realm.executeTransactionAsync()` tried to cal `onSucess` after the Realm was closed.